custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py

```diff
@@ -1357,48 +1717,64 @@ def add_arguments(self, parser):
         )
 
     def handle(self, domain, excel_file_path, format, **options):
-
         migration_id = self.generate_id()
         self.log_meta_info(migration_id, options['commit'])
-        column_mapping = self.get_column_mapping(format)
-        city_constants = self.get_city_constants(format)
+        import_format = self.get_import_format(format)
         case_factory = CaseFactory(domain)
 
         import_log_file_name = "drtb-import-{}.csv".format(migration_id)
+        bad_rows_file_name = "{}-bad-rows.csv".format(migration_id)
+        rows_with_unknown_exceptions = 0
 
         with open_any_workbook(excel_file_path) as workbook:
-            with open(import_log_file_name, "w") as import_log_file:
-                import_log_writer = csv.writer(import_log_file)
-                import_log_writer.writerow(["row", "case_ids", "exception"])
-
-                for i, row in enumerate(workbook.worksheets[0].iter_rows()):
-                    if i == 0:
-                        # Skip the headers row
-                        continue
-
-                    row_contains_data = any(cell.value for cell in row)
-                    if not row_contains_data:
-                        continue
-
-                    try:
-                        column_mapping.check_for_required_fields(row)
-                        case_structures = get_case_structures_from_row(
-                            domain, migration_id, column_mapping, city_constants, row
-                        )
-                        import_log_writer.writerow([i, ",".join(x.case_id for x in case_structures)])
-                        logger.info("Creating cases for row {}".format(i))
-
-                        if options['commit']:
-                            case_factory.create_or_update_cases(case_structures)
-                    except Exception:
-                        logger.info("Creating case structures for row {} failed".format(i))
-                        exception_as_string = traceback.format_exc()
-                        import_log_writer.writerow([i, "", exception_as_string])
+            with open(bad_rows_file_name, "w") as bad_rows_file:
+                with open(import_log_file_name, "w") as import_log_file:
```

Comment:
nit/tip: instead of using nested with statements, you can [open both files in the same `with` statement](https://docs.python.org/2/reference/compound_stmts.html#the-with-statement) and avoid the extra level of indentation.